drivers/modem/modem_cmd_handler.h

```diff
@@ -271,18 +277,25 @@ int modem_cmd_handler_init(struct modem_cmd_handler *handler,
  * @brief  Lock the modem for sending cmds
  *
  * This is semaphore-based rather than mutex based, which means there's no
- * requirements of thread ownership for the user. These functions are useful
+ * requirements of thread ownership for the user. This function is useful
  * when one needs to prevent threads from sending UART data to the modem for an
  * extended period of time (for example during modem reset).
  *
  * @param  *handler: command handler to lock
- * @param  lock: set true to lock, false to unlock
  * @param  timeout: give up after timeout
  *
  * @retval 0 if ok, < 0 if error.
  */
 int modem_cmd_handler_tx_lock(struct modem_cmd_handler *handler,
 			      k_timeout_t timeout);
+
+/**
+ * @brief  Unlock the modem for sending cmds
+ *
+ * @param  *handler: command handler to unlock
+ *
+ * @retval None.
```

Comment:
If the function does not return anything, the "retval"/"return" part of the docstring is typically skipped.